### PR TITLE
Portfolio Website: Finish Mobile Version

### PR DESCRIPTION
**Pull request by Megagig**
In this milestone, I implemented the following requirements
 I used html and css to create my portfolio website
 I installed linters configurations and made sure there are no linters error
 Build the work sections of the mobile website using the template chosen in the previous step.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,58 +7,268 @@
     <link rel="stylesheet" href="Styles.css" />
```
Comment (MeqdamAlqudah) [inline index.html:7]: - [x] Please make sure that the background of the toolbar section matches the design currently there is a difference between the background in the design and in your code.
Figma
image
Your design 
image

File: index.html [hunk 2]
```diff
@@ -7,58 +7,268 @@
...
     <title>Portfolio Website:Anthony</title>
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please fix this weird space behind the headline section.
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Megagig 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image
## Highlights
:+1: semantic tags 
:+1: design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- fixing the distance of behind the headline section by fixing the width of cards in the works section

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (joskalenda) [inline index.html]: - [x] This issue is not fixed yet, please take time to address all the required changes the previous reviewer mentioned

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Megagig , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @Megagig ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
 - No linter errors :+1: 
 - Good use of GitHub flow :+1: 
 - Responsive design :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**